.env.vagrant

```diff
@@ -33,6 +33,8 @@ OAUTH2_TOKEN_URI=http://web.chs-dev.internal:4000/oauth2/token
 OAUTH2_CLIENT_ID=
 OAUTH2_CLIENT_SECRET=
 
+COMPANY_AUTH_FEATURE_FLAG=0
```

Comment:
May I suggest different pattern? Out API used `CHIPS_FEATURE_ENABLED` which is bit more readable to me. Why not to have `COMPANY_AUTH_FEATURE_ENABLED` or `COMPANY_AUTH_VERIFICATION_FEATURE_ENABLED`

Names should be as clear as possible cause they will be maintained by people outside of the project.